docs/self-hosted-appcircle/installation-podman.md

```diff
@@ -0,0 +1,990 @@
+    │   ├── postgres
+    │   ├── report
+    │   ├── rijndael
+    │   ├── signing-identity
+    │   ├── store
+    │   ├── store-submit
+    │   ├── tester-web
+    │   ├── vault
+    │   ├── webApp
+    │   └── webhook
+    ├── generated-secret.yaml
+    ├── global.yaml
+    └── user-secret
+```
+
+Run Appcircle server services.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" up
+```
+
+:::info
+
+#### Artifact Registry Credentials: Cred.json
+
+Before we run self-hosted appcircle, we need to set artifact registry credentials. Using credentials JSON key file, we will pull container images for Appcircle server services.
+
+Although it's not required immediately at configuration steps, it's required while we're starting Appcircle server. Otherwise it can not pull container images from our artifact registry.
+
+For this reason, it's a part of the configuration. `ac-self-hosted.sh` bash script configures podman with appropriate credentials. If you don't have the key file, bash script gives error with a detailed message about the requirement.
+
+When you buy an enterprise license for self-hosted appcircle, you will get a credentials JSON key file which enables you to login our artifact registry. For example, assume our fictive company is Space Tech.
+
+You've got `space-tech-cred.json` key file and downloaded it into `~/Downloads` folder.
+
+First you need to copy that key file into self-hosted Appcircle root directory.
+
+```bash
+cp ~/Downloads/space-tech-cred.json cred.json
+```
+
+After that you can execute below command.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" up
+```
+
+You should see
+
+> "Podman login cred not found. Trying to login now..."
+
+in command output and then
+
+> "Login Succeeded"
+
+which shows us successful artifact registry login.
+
+If you get any error for some reason at this step, you can remove `${XDG_RUNTIME_DIR}/containers/auth.json` file to reset and execute same command again to retake same steps.
+
+:::
+
+:::info
+
+At first run, it will pull once all container images needed from artifact registry. Also there are some migration steps taken for once which prepares system for other operations.
+
+So it may need up to ~20 min to system be up according to your internet connection speed and CPU power. But recurrent boots will take a couple of minutes and will have shorter durations.
+
+:::
+
+Now you can check system health and gain an overview of the status.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" check
+```
+
+It will give a quick health summary. You should see below message on success.
+
+```
+ All services are running successfully. Project name is spacetech
+```
+
+If you want to get details about podman services, you may run the following commands.
+
+```bash
+cd projects/spacetech/export
+podman-compose ps
+```
+
+If everything is okay, then you should see service statuses as "running", "running (healthy)" or "exited (0)".
+
+![](https://cdn.appcircle.io/docs/assets/be-962-docker-compose-ps.png)
```

Comment:
Remove docker compose output.

```suggestion
```

Reply:
removed

Reply:
We should also remove the image link in Podman page.